flashlight/fl/test/nn/ModuleTest.cpp

```diff
@@ -22,6 +22,34 @@ namespace {
 
 class ContainerTestClass : public Sequential {
  public:
+  ContainerTestClass() = default;
+  ContainerTestClass(const ContainerTestClass& other) {
+    copy(other);
+  }
+  ContainerTestClass& operator=(const ContainerTestClass& other) {
+    copy(other);
+    return *this;
+  }
+  ContainerTestClass(ContainerTestClass&& other) = default;
+  ContainerTestClass& operator=(ContainerTestClass&& other) = default;
+  void copy(const ContainerTestClass& other) {
+    auto orphanParamIdxMap = other.getOrphanedParamsIdxMap();
+    for (int i = -1; i < static_cast<int>(other.modules_.size()); ++i) {
```

Comment:
`i` needs to be a negative for this loops to function as intended, hence I believe it needs to be `int`. In the future if updating to C++20, this could be changed to use `ssize()`.